tox.ini

```diff
@@ -0,0 +1,37 @@
+[tox]
+envlist = py26,py27,pypy,ppy32,py33,py34,pypy3,pep8
```

Comment:
Hypothesis doesn't officially support about half of these FWIW (and flat out doesn't support py26). The officially supported versions are 2.7, 3.3 and 3.4